docs/en/stack/ml/nlp/ml-nlp-model-ref.asciidoc

```diff
@@ -86,6 +154,26 @@ Using `DPREncoderWrapper`:
 * https://huggingface.co/nateraw/bert-base-uncased-emotion
 * https://huggingface.co/cardiffnlp/twitter-roberta-base-sentiment
 
+[discrete]
+=== Text classification expected model output
+
+With text classification, the entire sequence is classified. An example of this is sentiment analysis.
```

Comment:
IMO the example here doesn't add a lot of value, but if we keep it, I recommend merging the sentences like this:
```suggestion
With text classification (for example, in tasks like sentiment analysis), the entire sequence is classified.
```